flang/lib/Lower/OpenMP/DataSharingProcessor.cpp

```diff
@@ -143,7 +143,7 @@ void DataSharingProcessor::collectOmpObjectListSymbol(
 }
 
 void DataSharingProcessor::collectSymbolsForPrivatization() {
-  bool hasCollapse = false;
+
```

Comment:
super nit: an empty line at the start of a function looks wrong to me (feel free to ignore)